cyral/resource_cyral_sidecar_listener.go

```diff
@@ -352,7 +378,7 @@ func getSidecarListenerSchema() map[string]*schema.Schema {
 			Optional:    true,
 			// Notice the MaxItems: 1 here. This ensures that the user can only specify one this block.
 			MaxItems:      1,
-			ConflictsWith: []string{MySQLSettingsKey, DynamoDbSettingsKey},
+			ConflictsWith: []string{MySQLSettingsKey, DynamoDbSettingsKey, SQLServerSettingsKey},
```

Comment:
Should we also update the `ConflictsWith` in `DynamoDbSettingsKey`?
```Go
ConflictsWith: []string{S3SettingsKey, MySQLSettingsKey, SQLServerSettingsKey}
```

Reply:
yes, sorry. Fixed